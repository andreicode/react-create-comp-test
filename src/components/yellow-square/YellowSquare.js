import React from 'react';
import './style.scss';

const YellowSquare = (props) => {
  return (
    <div className="container">
      <p>YellowSquare</p>
    </div>
  );
};

export default YellowSquare;
