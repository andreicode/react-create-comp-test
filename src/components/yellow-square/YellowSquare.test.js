import React from "react";
import { render } from "@testing-library/react";
import YellowSquare from "./YellowSquare";

describe("[COMPONENT] YellowSquare", () => {
  it('renders correctly', () => {
    const { container } = render(<YellowSquare />);
    expect(container).toMatchSnapshot();
  });
});
