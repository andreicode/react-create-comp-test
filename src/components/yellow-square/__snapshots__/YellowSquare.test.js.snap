// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT] YellowSquare renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <p>
      YellowSquare
    </p>
  </div>
</div>
`;
